Makefile

```
@@ -102,8 +100,7 @@ image-upstream:
 	docker build -f upstream-example.Dockerfile .
 
 .PHONY: lint
-#lint:
-#	find . -type f -name '*.go' ! -name '*.pb.go' -print0 | xargs -0 goimports -w
+GOLANGCI_LINT_ARGS := --build-tags=$(TAGS)
```

When I was adding the linter to this repo I was making use of an externally-declared GOLANGCI_LINT_ARGS.  Can we either
```suggestion
GOLANGCI_LINT_ARGS += --build-tags=$(TAGS)
```
or
```suggestion
GOLANGCI_LINT_ARGS := "$(GOLANGCI_LINT_ARGS) --build-tags=$(TAGS)"
```

---

Ah, I was wondering why I wasn't seeing that anywhere. Yeah, I can make this change.